### ISSUE TITLE
Use the same symbol icons for the blob view sidebar and results of type 'symbol'

### ISSUE THREAD

**Issue by taras-yemets** [sourcegraph/sourcegraph#44946]
The symbol icons used for the `type:symbol` results page are different from what we use in the blob view symbols sidebar.
This may be a violation of WCAG [3.2.4: Consistent Identification](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html).
Check out https://github.com/sourcegraph/sourcegraph/issues/43154 for more details.

| Hierarchical Symbols | Results `type:symbol` | 
| -- | -- | 
| <img width="1028" alt="Screenshot 2022-11-29 at 11 04 31" src="https://user-images.githubusercontent.com/25318659/204486031-26904513-029e-4528-9ff0-6c4d0de9988a.png">|<img width="994" alt="Screenshot 2022-11-29 at 11 05 06" src="https://user-images.githubusercontent.com/25318659/204486043-4ad6d7a3-fd29-4127-913f-07322dabeba7.png">|

/cc @felixfbecker

**Comment by limitedmage** [comment]
Whoops, is https://github.com/sourcegraph/sourcegraph/issues/45076 a duplicate of this?

**Comment by felixfbecker** [comment]
(renamed the other issue to distinguish "making it consistent" from "adopting letter-based icons")
